### PR TITLE
Add regionalised benefits and Aus partner offer

### PR DESCRIPTION
**Pull request by andrewHEguardian**
### Current situation/background

We've added a new regionalised benefit to the checkout (https://github.com/guardian/support-frontend/pull/6163)

### What does this PR change?

This PR adds the same capability and specific new benefit to MMA

UK
image

Aus
image

### CODE REVIEW

File: client/components/mma/shared/benefits/BenefitsConfiguration.ts [hunk 1]
```diff
@@ -27,10 +31,37 @@ const guardianWeekly = {
...
+		specificToRegions?: CurrencyIso[];
+	},
+	currencyIso: string,
+) {
+	if (isCurrencyIso(currencyIso)) {
+		if (benefit.specificToRegions !== undefined) {
+			return benefit.specificToRegions.includes(currencyIso);
+		}
+
+		return true;
```
Comment (rBangay) [inline client/components/mma/shared/benefits/BenefitsConfiguration.ts:58]: Not sure if this is more succinct or if keeping the if statement is clearer 🤷 
```
return benefit.specificToRegions ? benefit.specificToRegions.includes(currencyIso) : true;
```

**Comment by rBangay** [comment]
Happy with everything technically, just a quick question around design on mobile breakpoints:

Screenshot 2024-07-21 at 22 06 11

The existing benefits have the description as a seperate sentence (they start with a capitol letter) which makes a bit more sense on mobile breakpoints .. this new one doesn't ... Sorry if design have already been over this, I've approved either way 👍

**Review by rBangay — APPROVED** [review]

File: client/components/mma/shared/benefits/BenefitsConfiguration.ts (hunk 1 above)
Comment (andrewHEguardian) [reply client/components/mma/shared/benefits/BenefitsConfiguration.ts:58]: had the same thought, erred on the side of clarity

**Comment by andrewHEguardian** [comment]
> Happy with everything technically, just a quick question around design on mobile breakpoints:
> 
> Screenshot 2024-07-21 at 22 06 11
> The existing benefits have the description as a seperate sentence (they start with a capitol letter) which makes a bit more sense on mobile breakpoints .. this new one doesn't ... Sorry if design have already been over this, I've approved either way 👍

thanks that's a great shout, I didn't consider this

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**